pkg/controller/node/controller_utils.go

```diff
@@ -164,13 +164,13 @@ func (nc *NodeController) maybeDeleteTerminatingPod(obj interface{}) {
 	// guarantee backwards compatibility of master API to kubelets with
 	// versions less than 1.1.0
 	node := nodeObj.(*api.Node)
-	v, err := version.Parse(node.Status.NodeInfo.KubeletVersion)
+	v, err := utilversion.ParseSemantic(node.Status.NodeInfo.KubeletVersion)
 	if err != nil {
 		glog.V(0).Infof("couldn't parse verions %q of minion: %v", node.Status.NodeInfo.KubeletVersion, err)
 		utilruntime.HandleError(nc.forcefullyDeletePod(pod))
 		return
 	}
-	if gracefulDeletionVersion.GT(v) {
+	if !v.AtLeast(gracefulDeletionVersion) {
```

Comment:
This would read better as `v.LessThan(gracefulDeletionVersion)`, but maybe there's a reason for this semantic instead?